src/main/pages/user-management/authentication.md

```diff
@@ -1,16 +1,19 @@
 ---
-title: "Keycloak Authentication"
-keywords: organizations, user management, permissions, authentication
-tags: [ldap, keycloak]
+title: "Che Authentication"
+keywords: user management, permissions, authentication
+tags: [keycloak]
 sidebar: user_sidebar
 permalink: authentication.html
 folder: user-management
 ---
 {% include links.html %}
 
-## Auth Filters on master
+## Authentication on Che Master
+
+### Auth Filters on master
+
  Che master can handle both types of signed requests - using Keycloak token or Machine token.
- Authentication tokens can be send either in a `Authorization` header or `token` query param.
+ Authentication tokens can be send in a `Authorization` header. Also, in limited cases when it's not possible to use `Authorization` header, token can be send in `token` query parameter. An example of it is initialization iframe where authentication is needed, IDE shows java doc in this way.
```

Comment:
An example of such exceptional case can be: OAuth authentification initialization, IDE shows **javadoc** ...